docs/usage.md

```diff
@@ -74,7 +74,14 @@ The tool offers also the ability to flash SSL certificates to a module:
 ./arduino-fwuploader certificates flash -b arduino:samd:nano_33_iot" -a COM10 -u arduino.cc:443 -u google.com:443
 ```
 
-or you can specify a path to a file with `-f` instead of the URL of the certificate
+or you can specify a path to a file in `.der` format with `-f` instead of the URL of the certificate
+
+Due to a limitation on the handling of certs in the Nina modules, when `certificates flash` command is launched, all the
+previous certificates are going to be erased. To overcome this it's recommended to upload them all together:
```

Comment:
```suggestion
previous certificates are going to be erased. To overcome this it's required to upload them all together:
```
"Recommended" implies this is a best practice, with some other inferior option available. Unless I misunderstand, this is the only way to overcome it, so "required" is the appropriate description.